woocommerce/woocommerce-ios
# Products: Only use one bottom sheet for both AI and manual product type creation

Part of: #11361


## Description


This PR is part one to do the improvement listed in #11361.

It deals with displaying the list of product type creation options immediately when "Add manually" is selected in the product creation AI bottom sheet, instead of opening to another sheet.

The design is not yet identical to the video in #11361, it shall be done separately. Specifically, these are not added yet:
- Use new heading "Create Product / select product type")
- Split list of product type creation options into sub-lists (Standard, Subscription, Others).

## Testing instructions


1. In a site that's eligible for AI support, go to Products tab,
2. Press "+",
3. On the bottom sheet that appear, tap "Add manually",
4. Ensure the product types creation options are shown.
5. Try an option and ensure it works properly.
6. Also check that analytics are tracked properly for each option selection: `add_product_product_type_selected`

## Video


https://github.com/woocommerce/woocommerce-ios/assets/266376/5258423c-8292-4fe5-8e5f-8ee84d3ff74c



---
